README.md

```diff
@@ -339,6 +339,7 @@ that caused Neoformat to be invoked.
 - Lua
   - [`luaformatter`](https://github.com/LuaDevelopmentTools/luaformatter)
   - [`lua-fmt`](https://github.com/trixnz/lua-fmt)
+  - [`lua-format`](https://github.com/Koihik/LuaFormatter)
```

Comment:
looks good!

Could you also update the vim docs at `docs/neoformat.txt`?

Reply:
Done~
Added it to `docs/neoformat.txt` also.